ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c

```diff
@@ -402,11 +402,11 @@ nnfw_convert_to_gst_info (const nnfw_tinfo_s * nnfw_info,
     if ((status = nnfw_tensor_type_to_gst (ninfo->dtype, &ginfo->type)) != 0)
       return status;
 
+    for (idx = 0; idx < NNS_TENSOR_RANK_LIMIT; idx++)
```

Comment:
Actually, this is unnecessary. `gst_tensors_info_init (gst_info);` at line 392 would initialize dim with 0.

Reply:
Thank you :)
I removed this part